glibc.yaml

```diff
@@ -120,6 +120,10 @@ pipeline:
     with:
       patches: Disable-AVX512VL.patch
 
+  - runs: |
+      gccdir=$(GCC_SPEC_FILE=/dev/null gcc --print-search-dirs | grep ^install: | cut -d' ' -f2)
+      sed -r 's/-z now//' < "$gccdir/openssf.spec" > /home/build/openssf.spec
+
```

Comment:
if we are confident that gcc-15 fixed it, should we drop this hunk altogether and use the default build?